In [1]:
#1~3셀 매장 가구 항목 및 가격 설정
lovely_loveseat_description = "Lovely Loveseat. Tufted polyester blend on wood. 32 inches high x 40 inches wide x 30 inches deep. Red or white."
lovely_loveseat_price = 254.00

In [5]:
stylish_settee_description = "Stylish Settee. Faux leather on birch. 29.50 inches high x 54.75 inches wide x 28 inches deep. Black."
stylish_settee_price = 180.50

In [7]:
luxurious_lamp_description = "Luxurious Lamp. Glass and iron. 36 inches tall. Brown with cream shade."
luxurious_lamp_price = 52.15

In [9]:
#매출세 8.8% 설정
sales_tax = 0.088

In [11]:
#고객 정보 초기화
customer_one_total = 0
customer_one_itemization = " "

In [13]:
#고객 구매 정보
customer_one_itemization += lovely_loveseat_description #가구 설명
customer_one_total += lovely_loveseat_price

In [15]:
#추가 구매 
customer_one_itemization += luxurious_lamp_description
customer_one_total += luxurious_lamp_price

In [17]:
#세금 
customer_one_tax = customer_one_total * sales_tax
customer_one_total += customer_one_tax

In [19]:
#영수증 출력
print("Customer One Items:")
print(customer_one_itemization)

print("Customer One Total:")
print(customer_one_total)

Customer One Items:
 Lovely Loveseat. Tufted polyester blend on wood. 32 inches high x 40 inches wide x 30 inches deep. Red or white.Luxurious Lamp. Glass and iron. 36 inches tall. Brown with cream shade.
Customer One Total:
333.09119999999996


예상출력 비교
Customer One Items:
Lovely Loveseat. Tufted polyester blend on wood. 32 inches high x 40 inches wide x 30 inches deep. Red or white.Luxurious Lamp. Glass and iron. 36 inches tall. Brown with cream shade.
Customer One Total:
333.09119999999996